ci/setup-tests.ps1

```diff
@@ -0,0 +1,21 @@
+# Expects gdk_home
+param(
+    [string] $build_output_dir,
+    [string] $project_path,
+    [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine"
+)
+
+# Copy the built files back into the SpatialGDK folder, to have a complete plugin
+# The trailing \ on the destination path is important!
+Copy-Item -Path "$build_output_dir\*" -Destination "$gdk_home\SpatialGDK\" -Recurse -Container -ErrorAction SilentlyContinue
```

Comment:
For this and the following ps commands - are we fine continuing if they fail (`-ErrorAction SilentlyContinue`)?

Reply:
The silent continue is there to ignore any errors thrown when attempting to copy over files that already exist at the target location. Since we're merging folders, in local testing I've found that to be necessary. I think this should be fine since we'll fail later if the assembled GDK is somehow broken, otherwise an alternative would be to somehow overwrite existing files.